ddtrace/internal/writer/writer.py

```diff
@@ -383,9 +383,9 @@ def flush_queue(self, raise_exc=False):
 
     def _flush_queue_with_client(self, client, raise_exc=False):
         # type: (WriterClientBase, bool) -> None
-        n_traces = len(client.encoder)
+        n_traces = 0
```

Comment:
This fixes a broken test:
```suggestion
        n_traces = len(client.encoder)
```